Documentation/csi_rn_knownissues.md

```diff
@@ -0,0 +1,22 @@
+# Known issues
```

Comment:
do we plan to maintain all this PR content from now on, in addition to the knowledge center?
I'm not sure all/any of the duplications are necessary

Reply:
anything i have here i can have and plan on also on IBM Documentation
the only thing that might change is perhaps the PDFs of the content. I am working on that.
FYI @erantzabari